pkg/controllers/nodeclaim/consistency/controller.go

```diff
@@ -102,7 +102,7 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeClaim
 			return reconcile.Result{}, fmt.Errorf("checking node with %T, %w", check, err)
 		}
 		for _, issue := range issues {
-			logging.FromContext(ctx).Errorf("check failed, %s", issue)
+			log.FromContext(ctx).Error(err, "Consistency error")
```

Comment:
same here on decapitalization
```suggestion
			log.FromContext(ctx).Error(err, "consistency error")
```